clients/java/signalr/build.gradle

```diff
@@ -16,3 +17,32 @@ dependencies {
     implementation "org.java-websocket:Java-WebSocket:1.3.8"
     implementation 'com.google.code.gson:gson:2.8.5'
 }
+
+task sourceJar(type: Jar) {
+    classifier "sources"
+    from sourceSets.main.allJava
+}
+
+task javadocJar(type: Jar, dependsOn: javadoc) {
+    classifier "javadoc"
+    from javadoc.destinationDir
+}
+
+task generatePOM {
+    pom {
+        project {
+            groupId 'com.microsoft.aspnetcore'
+            artifactId 'signalr'
+            version '0.1.0'
+
+            inceptionYear '2018'
```

Comment:
![Top Spinning from the movie "Inception"](https://media.giphy.com/media/z1meXneq0oUh2/giphy.gif)